portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +32,65 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
+    private int maxDisplay = 5;
```

Comment:
private static final int MAX_DISPLAY = 5;

Style for defining a constant:
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
this variable actually does get changed later and 5 is the initial default value:)